### PR TITLE
Reorder facets, fixes #136

### LINKED ISSUE #136: Change order of facets
To: 

Format/Genre
Date of Publication
Place of Publication 
Subject: Topic
Subject: Geographic
Language

### PR DESCRIPTION
**Pull request by hackmastera**

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -77,13 +77,13 @@ class CatalogController < ApplicationController
...
-    config.add_facet_field 'language_label_facet', label: 'Language'
-    config.add_facet_field 'geographic_subject_label_facet', label: 'Geographic Subject'
-    config.add_facet_field 'geographic_origin_label_facet', label: 'Geographic Origin'
     config.add_facet_field 'date_created_facet', label: 'Date Created', range: true
+    config.add_facet_field 'geographic_origin_label_facet', label: 'Geographic Origin'
     config.add_facet_field 'category_subject_facet', label: 'Subjects', pivot: ['category_facet', 'subject_label_facet']
+    config.add_facet_field 'geographic_subject_label_facet', label: 'Geographic Subject'
+    config.add_facet_field 'language_label_facet', label: 'Language'
+    config.add_facet_field 'category_facet', label: 'Category', show: false
+    config.add_facet_field 'subject_label_facet', label: 'Subject', show: false
```
Comment (escowles) [inline app/controllers/catalog_controller.rb]: Do we want these separate from the Category/Subject pivot facet above?

Comment (hackmastera) [reply app/controllers/catalog_controller.rb]: Yeah I am not sure; they're `show:false` so I'm not sure how they're being used exactly.

Comment (tpendragon) [reply app/controllers/catalog_controller.rb]: I think it's an implementation trick to get the pivot facet to work?

Comment (escowles) [reply app/controllers/catalog_controller.rb]: 👍  that makes sense.

**Pull request merged**